src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -1,27 +1,28 @@
 import datetime as dt
 from abc import ABC, abstractmethod
 from pathlib import Path
+from typing import Any, Dict
 
 import numpy as np
 import pandas as pd
 from loguru import logger
+from vivarium import Component
 from vivarium.framework.engine import Builder
 from vivarium.framework.event import Event
-from vivarium.framework.population import PopulationView
 from vivarium_public_health.utilities import DAYS_PER_YEAR
 
 from vivarium_census_prl_synth_pop import utilities
 from vivarium_census_prl_synth_pop.constants import data_values, metadata, paths
 
 
-class BaseObserver(ABC):
+class BaseObserver(Component):
```

Comment:
If we want to continue inheriting from `ABC`, you can do `BaseObserver(ABC, Component):` (I think ABC has to be first, but I can't remember).

If you're not going to use ABC you can remove the ABC imports.